share/spack/gitlab/cloud_pipelines/stacks/gcp-base/spack.yaml

```diff
@@ -0,0 +1,24 @@
+spack:
+  view: false
+  definitions:
+  - arch:
+    - "%gcc"
+
+  - packages:
+    - "stream"
+
+  - targets:
+    - "target=x86_64_v3"
+
+  specs:
+  - matrix:
+    - - $packages
+    - - $arch
+    - - $targets
+  ci:
+    pipeline-gen:
+    - build-job:
+        image: "ghcr.io/spack/e4s-centos-7:v2023-10-30"
```

Comment:
This still needs the tags from before, these are applied to the build jobs and without them these will get scheduled on non-gcp runners